modules/osdk-upgrading-v130-to-v180.adoc

```diff
@@ -0,0 +1,895 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-upgrading-projects.adoc
+
+:osdk_ver: v1.8.0
+:osdk_ver_n1: v1.3.0
+
+[id="osdk-upgrading-v130-to-v180_{context}"]
+= Upgrading projects for Operator SDK {osdk_ver}
+
+The following upgrade steps must be performed to upgrade an existing Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+- Operator SDK {osdk_ver} installed
+- Operator project that was previously created or maintained with Operator SDK {osdk_ver_n1}
+
+.Procedure
+
+. Make the following changes to your `PROJECT` file:
+
+.. Update the `PROJECT` file `plugins` object to use `manifests` and `scorecard` objects.
++
+The `manifests` and `scorecard` plug-ins that create Operator Lifecycle Manager (OLM) and scorecard manifests now have plug-in objects that direct `create` subcommands to create related files.
+
+** For Go-based Operator projects, an existing Go-based plug-in configuration object is already present. While the old configuration is still supported, these new objects will be useful in the future as configuration options are added to their respective plug-ins:
++
+.Old configuration
+[source,yaml]
+----
+version: 3-alpha
+...
+plugins:
+  go.sdk.operatorframework.io/v2-alpha: {}
+----
++
+.New configuration
+[source,yaml]
+----
+version: 3-alpha
+...
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+----
+
+** Optional: For Ansible- and Helm-based Operator projects, the plug-in configuration object previously did not exist. While not required to add, these new objects will be useful in the future as configuration options are added to their respective plug-ins:
```

Comment:
nit again: "While you are not required to add the plug-in configuration objects, these new..."?